src/components/MessageBox/index.tsx

```diff
@@ -18,6 +25,22 @@ type SubTabType = {
   hoverClass: string;
 };
 
+const getTabs = (
+  ownerRequest?: OwnerRequestType[],
+  userRequests?: UserRequestType[],
+): Tab[] => {
+  const tabs: (Tab | null)[] = [
+    ownerRequest?.length
+      ? { value: 'received', label: 'Příchozí žádosti', items: ownerRequest }
+      : null,
```

Comment:
If you would do

const tabs: (Tab | null)[] = [
    ...(ownerRequest?.length
      ? { value: 'received', label: 'Příchozí žádosti', items: ownerRequest }
      : []),
      
      
      you can get rid of the filtering :))